File: round 1/RoSo/problem2.java
import java.util.*;
class problem2
{
boolean isprime(int n)
{
int c=1;
for(int i=2;i<=n;i++)
{
if(n%i==0)
{
c=c+1;
}
}
if(c==2)
return (true);
else
return (false);
}

public static void main(String args[])
{
problem2 ob=new problem2();
Scanner sc=new Scanner(System.in);
int l, count=0;
String n=sc.nextLine();
l=n.length();
for (int i=0; i<l; i++)
{
char c=n.charAt(i);
int j=(int)c;
if(ob.isprime(j)==true)
count++;
}
System.out.println(count);
}
}

File: round 1/Abhra_Anindita/problem2.c
#include<stdio.h>
int isPrime(int);
main()
{
char str[100];
int i,val,count=0;
for(i=0;i<100;i++)
{
val=str[i];
if(isPrime(val)==1)
count=count+1;
}
printf("%d",count);
}
int isPrime(int n)
{
int i,c;
for(i=2;i<=n;i++)
{
if(n%i==0)
c=c+1;
}
if(c==1)
return 1;
else
return 0;
}